srtp_cipher.go

```diff
@@ -6,6 +6,7 @@ import "github.com/pion/rtp"
 // of the SRTP Specific ciphers
 type srtpCipher interface {
 	authTagLen() int
```

Comment:
https://tools.ietf.org/html/rfc3711#section-5.2
>   HMAC-SHA1, as defined in Section 4.2.1, SHALL be the default message
>   authentication code.  The default session authentication key-length
>   (n_a) SHALL be 160 bits, the default authentication tag length
>   (n_tag) SHALL be 80 bits, and the SRTP_PREFIX_LENGTH SHALL be zero
>   for HMAC-SHA1.  In addition, for SRTCP, the pre-defined HMAC-SHA1
>   MUST NOT be applied with a value of n_tag, nor n_a, that are smaller
>   than these defaults.  For SRTP, smaller values are NOT RECOMMENDED,
>   but MAY be used after careful consideration of the issues in Section
>   7.5 and 9.5.

In my understanding, RFC only specifies default value.
User can select shorter/longer auth tag len (e.g. 32-bit tag is pre-defined) independent from the algorithm defiintion on HMAC-SHA1. (On GCM, it must be always 16 bytes.)

Reply:
Yea! So `srtp_cipher_aes_cm_hmac_sha1` should be a little more general for `ProtectionProfileAes128CmHmacSha1_32`. I can make that change, but I don't think it is worth it (for WebRTC users)

FireFox dropped support (might not be done yet) for `SRTP_AES128_CM_HMAC_SHA1_32` and I think everyone else followed.

Reply:
For WebRTC case, yes. But for more general IoT usecases, I think 32-bit tag is still worth. (https://tools.ietf.org/html/rfc3711#section-7.5)

Reply:
:+1: Yea I am totally open to it being added! I don't plan on doing the work right now though (unless we have a user that is blocked)

Reply:
@at-wat Do you want me to implement that in this PR? Or can I create a follow up issue?

Reply:
I think it's fine to be a follow up issue!